docs/docs/troubleshooting.md

```diff
@@ -33,3 +33,35 @@ To make it easier for us to understand your issue and to be able to approach it
 - `Native` gesture may not conform to the standard state flow due to platform specific workarounds to incorporate native views into RNGH.
 - Keep in mind that `Touchables` from RNGH are rendering two additional views that may need to be styled separately to achieve desired effect (`style` and `containerStyle` props).
 - In order for the gesture composition to work, all composed gestures must be attached to the same `GestureHandlerRootView`.
+
+### Multiple versions of Gesture Handler were detected
+
+This error usually happens when in your project there exists more than one instance of Gesture Handler. It can occur when some of your dependency has installed Gesture Handler inside their own `node_modules` instead of using it as a peer dependency. In this case two different versions of Gesture Handler JS module try to install the same Native Module. You can resolve this problem manually by modifying your `package.json` file.
```

Comment:
```suggestion
This error usually happens when in your project there exists more than one instance of Gesture Handler. It can occur when some of your dependencies have installed Gesture Handler inside their own `node_modules` instead of using it as a peer dependency. In this case two different versions of Gesture Handler JS module try to install the same Native Module. You can resolve this problem manually by modifying your `package.json` file.
```